src/server/controllers/ApiController.js

```diff
@@ -1,5 +1,26 @@
+import User from '../models/user';
+
 export default class ApiController {
   static index (req, res) {
     res.json({ version: '0.0.1' });
   }
+
+  static postAddUser (req, res, next) {
+    const firstname = req.body.firstname;
```

Comment:
This all constants can be distrusted from req.body, like this 
Const { firstname, email, etc.. } = req.body

Reply:
changed